com/win32comext/authorization/demos/EditServiceSecurity.py

```diff
@@ -1,7 +1,7 @@
 """
 Implements a permissions editor for services.
 Service can be specified as plain name for local machine,
-or as a remote service of the form \\machinename\service
+or as a remote service of the form \\machinename\\service
```

Comment:
I'm quite surprised this tool is so opinionated about comments, but it got it wrong here - the existing "\\\\" should now be "\\\\\\\\"

Reply:
I don't think it's an opinion about comments: it correctly found (and fixed) an invalid escape sequence: `\s`. It didn't touch the existing `\\` because that"s a valid `\` character once escaped.

That being said, you're right this docstring should have `\\\\machine` to properly escape to `\\machine`